### DISCUSSION TITLE
Stops and Routes vector tiles return 204 HTTP code starting from zoom level 15

### DISCUSSION THREAD

**Discussion by dmitrykinakh** [transitland/transitland#416] [Q&A] [answered]
Hi community,

In my application, I'm trying to integrate stops and routes vector tiles from Transit Land per https://www.transit.land/documentation/vector-tiles#mapbox-gl-example guide. I was able to connect them, however, I've noticed that I'm getting 204 HTTPS responses if I try to fetch stops/routes starting from the zoom level 15 and deeper.

Is there any reason why Transit Land does now have those areas generated?
![image](https://github.com/transitland/transitland/assets/8025637/fb1724a3-813d-4ecf-84da-6d5cec8b2f0b)

Note: the same area on Z14 has tiles present
![image](https://github.com/transitland/transitland/assets/8025637/6048872c-f3ac-4536-8d81-e2106ea0c19c)

**Comment by drewda** [comment, marked as answer]
The Transitland v2 Vector Tiles for routes and for stops are both generated up to zoom level 14.

As included in the example code at https://www.transit.land/documentation/vector-tiles, you can set a "max zoom" value of 14 to ensure that Mapbox GL JS (or Maplibre or Leaflet) does not try to load and display tiles past that zoom level.

Some libraries can also do their own client-side interpolation at higher zoom levels. In Leaflet, set a `maxNativeZoom` value of 14 and then it will calculate how to display those tiles at higher zoom levels.

**Comment by dmitrykinakh** [comment]
> The Transitland v2 Vector Tiles for routes and for stops are both generated up to zoom level 14.

That piece of info was not specified in an obvious way but thanks for a quick reply. I've already adjusted my code so it shows stops and routes.
